### PR TITLE
Removes TEST from all DPR file names and updates Program statement

### PR DESCRIPTION
**Pull request by rpottsoh**
Because TEST was appearing in the filename the site would list the DPR as part of the test suite.  While the DPR file is necessary it is essentially boiler plate and not the actual unit that contains tests, so I would rather not see it listed in the web interface.

### CODE REVIEW

File: exercises/allergies/allergies.dpr [hunk 1]
```diff
@@ -1,4 +1,4 @@
-program AllergyTests;
+program allergies;
```
Comment (kotp) [inline exercises/allergies/allergies.dpr]: This one is not like the others.  `allergies` as opposed to `Allergies`

**Comment by rpottsoh** [comment]
I can take care of that.  I think it ended up lowercase because that is how it is listed in x-common.  I didn't really pay that close attention to the others.  Allergies was the first one. 

Sent from my android device.

-----Original Message-----

File: exercises/allergies/allergies.dpr (hunk 1 above)
Comment (rpottsoh) [inline exercises/allergies/allergies.dpr]: @kotp my 2nd and 3rd commits take care of the issue.  I tried squashing the 2nd and 3rd together but that kills the file rename and stuck with the original lowercase filename.  Hopefully when this PR is squashed and merged the rename takes.

**Comment by kotp** [comment]
You can squash the commits.  We don't need the first commit to be separate.  Looks good to me.

**Comment by rpottsoh** [comment]
Thanks, will do.


Sent from my android device.

**Pull request merged**